### PR TITLE
Update

### PR DESCRIPTION
**Pull request by IeuanWalker**
## Submission description
UI control to auto generate a breadcrumb navigation

### CODE REVIEW

File: src/Snppts/Snippets/BreadcrumbControl.cs [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    public class BreadcrumbControl : IAmASnippet
+    {
+        public string Slug => "breadcrumb-control";
+        public string Title => "Breadcrumb control";
+        public string GithubRepoName => "IeuanWalker/Maui.Breadcrumb";
+        public bool ContainsAndroidSample => true;
+        public bool ContainsiOSSample => true;
+        public bool ContainsUWPSample => false;
+
+        public string Description => "This is a breadcrumb navigation control https://www.surveymonkey.com/r/maui-ui-challengethat is completely automatic and uses the Navigation stack to get the page titles to generate the breadcrumbs.";
```
Comment (jfversluis) [inline src/Snppts/Snippets/BreadcrumbControl.cs]: This description seems a bit funky :D

**Review by jfversluis — CHANGES_REQUESTED** [review]
One thing here, the rest looks good!

File: src/Snppts/Snippets/BreadcrumbControl.cs (hunk 1 above)
Comment (IeuanWalker) [reply src/Snppts/Snippets/BreadcrumbControl.cs]: @jfversluis lol not sure how that happened, should be ok now

**Review by jfversluis — APPROVED** [review]

**Comment by jfversluis** [comment]
Thanks for this great contribution! I think you found the link to claim your stickers already 😜

**Pull request merged**